usaspending_api/disaster/v2/views/overview.py

```diff
@@ -95,26 +103,28 @@ def award_outlays(self):
             .aggregate(total=Sum("gross_outlay_amount_by_award_cpe"))["total"]
         ) or 0.0
 
-    def remaining_balances(self):
-        remaining_balances = list(
+    def total_obligations(self):
+        return (
             latest_gtas_of_each_year_queryset()
             .filter(disaster_emergency_fund_code__in=self.defc)
-            .values("fiscal_year")
-            .annotate(total=Sum("unobligated_balance_cpe"))
-            .values("fiscal_year", "total")
-            .order_by("-fiscal_year")
-        )
-
-        return remaining_balances[0]["total"] if remaining_balances else Decimal("0.0")
+            .values("obligations_incurred_total_cpe", "deobligations_or_recoveries_or_refunds_from_prior_year_cpe")
+            .aggregate(
+                total=(
+                    Sum("obligations_incurred_total_cpe")
+                    - Sum("deobligations_or_recoveries_or_refunds_from_prior_year_cpe")
```

Comment:
Should this not be a plus instead of a minus? Tony pushed a similar fix to the feature branch for some of the other endpoints: https://github.com/fedspendingtransparency/usaspending-api/pull/2990/files#diff-cce840c2a4e4c2aaebda1763a4740fed552088efa44e88163ef45f2a81b81e6cR156

Reply:
Disregard, different File type.